### PR TITLE
Organizers implementation for the initiative form.

### PR DESCRIPTION
**Pull request by nick22891**
This pull request addresses the issue with organizers added via the initiatives form not being reflected on the back-end.

### CODE REVIEW

File: models/Organizer.js [hunk 1]
```diff
@@ -4,7 +4,9 @@
 		Organizer = new keystone.List('Organizer');
 
 	Organizer.add({
-		name: {type: String}
+		name: {type: String},
+		text: {type:String},
```
Comment (matjames007) [inline models/Organizer.js]: What are the attributes `text` and `value`?

File: public/js/jquery.tokenize.js [hunk 2]
```diff
@@ -0,0 +1,861 @@
+/**
```
Comment (matjames007) [inline public/js/jquery.tokenize.js]: @nick remove this file from the repository.  Instead use bower to manage:
```
bower install jquery-tokenize --save
```

File: routes/api/organizer.js [hunk 3]
```diff
@@ -5,6 +5,38 @@
...
+			})
+			//.where('state', 'published')
+			//.populate('')
+			.exec(function(err, organizers){
+				
+				for (var i = 0;i < organizers.length; i++) {
+
+					organizers[i].text = organizers[i].name;
+
+					organizers[i].value = organizers[i]["_id"];
```
Comment (matjames007) [inline routes/api/organizer.js]: @nick22891 what are you doing?  Why did you add `text` and `value` to the model itself?  Do not mix the views on the frontend with that the logic of the backend.  Do the necessary transformations on the frontend.

Comment (nick22891) [inline routes/api/organizer.js]: @matjames007 I've made other edits as per your comments that will be reflected in the next commit but this particular implementation was necessary because the open source tokenizer plugin expects to receive "text" and "value" fields from the API endpoint (and this behavior is not configurable). Basically I'm obliged to pass it an url and it needs to receive JSON output with those specific field names to work. So doing it on the front-end wouldn't work since the tokenizer needs to interface directly with the endpoint.

Do you have any alternative ways for implementing this?

**Comment by nick22891** [comment]
This pull request still hasn't been merged. I see where @matjames007 raised some concerns that I tried to answer but I'm going to do another commit on this branch today to try to get it acceptable to be merged in. Until that happens Doc will still be seeing the bug with the organizers...

File: templates/views/layouts/default.hbs [hunk 4]
```diff
@@ -105,6 +107,19 @@
 			//- them specifically here, instead of bootstrap-3.3.5.min.js
 		--}}
 
+        <script type="text/javascript">
+
+            jQuery(document).ready(function() {
```
Comment (matjames007) [inline templates/views/layouts/default.hbs]: @nick22891 check this out to see if it makes sense.  I have not tested these changes!
